src/test/java/com/hotjava/app/hotjava/HotJavaApplicationUnitTests.java

```diff
@@ -0,0 +1,168 @@
+		assertEquals(VEHICLE_SCORE,mockVehicle.getVehicleScore());
+		assertEquals(mockPhoto, mockVehicle.getPhoto());
+	}
+
+	void givenMockVehicleAndPhotoWerePopulated(){
+		givenMockVehicleIsPopulated();
+		mockPhoto.setPhotoId(PHOTO_ID);
+		mockPhoto.setFileName(PHOTO_FILE_NAME);
+		mockPhoto.setPath(PHOTO_PATH);
+	}
+
+	void givenMockVehicleIsPopulated(){
+		mockVehicle.setVehicleSubmissionID(VEHICLE_ID);
+		mockVehicle.setVehicleOwnerName(VEHICLE_OWNER_NAME);
+		mockVehicle.setVehicleDescription(VEHICLE_DESCRIPTION);
+		mockVehicle.setVehicleYear(VEHICLE_YEAR);
+		mockVehicle.setVehicleMake(VEHICLE_MAKE);
+		mockVehicle.setVehicleModel(VEHICLE_MODEL);
+		mockVehicle.setVehicleScore(VEHICLE_SCORE);
+		mockVehicle.setPhoto(mockPhoto);
+	}
+
+	/**
+	 * Validate that a complete Vehicle form can be submitted with confirmation notification.
+	 */
+	@Test
+	void vehicleFormSubmissionWithAllData() throws Exception {
+		//TODO adjust method below after views and forms are created
+		givenVehicleDataIsAvailable();
+		whenCompleteVehicleFormIsSubmitted();
+		thenVehicleCanBeSavedWithConfirmationMessage();
+	}
+
+	private void givenVehicleDataIsAvailable() throws Exception {
+		Mockito.when(vehicleDAO.save(mockVehicle)).thenReturn(mockVehicle);
+	}
+
+	private void whenCompleteVehicleFormIsSubmitted() {
+		givenMockVehicleAndPhotoWerePopulated();
+	}
+
+	private void thenVehicleCanBeSavedWithConfirmationMessage() throws Exception {
+		//TODO adjust method below after views and forms are created
+		Vehicle createdVehicle = vehicleService.save(mockVehicle);
+		assertEquals(mockVehicle,createdVehicle);
+		verify(vehicleDAO, atLeastOnce()).save(mockVehicle);
+
+		//TODO verify confirmation
+	}
+
+	/**
+	 * Validate that an incomplete Vehicle can not be saved.
+	 */
+	@Test
+	void vehicleIncompleteDataCanNotBeStored() throws Exception {
+		givenVehicleDataIsAvailable();
+		whenIncompleteVehicleForm();
+		thenVehicleIsNotStored();
+	}
+
+	private void whenIncompleteVehicleForm() {
+		givenMockVehicleIsPopulated();
+	}
+
+	private void thenVehicleIsNotStored() throws Exception {
+		Vehicle rejectedVehicle = vehicleService.save(mockVehicle);
+		assertEquals(mockVehicle,rejectedVehicle);
+		//vehicleDAO should not be called
+		verify(vehicleDAO, never()).save(mockVehicle);
+
+	}
+
+	/**
+	 * Validate that vehicles score is updated when a vote is made.
+	 */
+	@Test
+	void vehicleScoreUpdatedWhenVoted(){
+		givenVehicleProfileIsPresent();
+		whenVoteIsSubmitted();
+		thenVehicleScoreIsUpdated();
+	}
+
+	private void givenVehicleProfileIsPresent() {
+		//TODO
+	}
+
+	private void whenVoteIsSubmitted() {
+		//TODO
+	}
+
+	private void thenVehicleScoreIsUpdated() {
+		//TODO
+	}
+
+	/**
+	 * Validate that new view with different vehicle are returned when a request to "skipp" a vehicle is made.
+	 */
+	@Test
+	void newVehicleWhenSkipped(){
+		givenVehicleProfileIsPresent();
+		whenSkipVehicleIsRequested();
+		thenNewViewWithDiffrentVehicleIsReturned();
+	}
+
```

Comment:
This is an example where I renamed the test, the Java Docs, and some of the called methods to change the meaning of it from UI test to unit testing.
My goal was not to provide the implementation of the tests that the group created, but rather rename thing (only) to remove any UI testing plans form the testing/ unit testing file.